MAUI/Chat/messages.md

```diff
@@ -1366,6 +1366,28 @@ By default, the author’s name and avatar are displayed for the incoming messag
 
 ![Hide incoming avatar and author visibility in .NET MAUI Chat](images/messages/maui-chat-hide-avatar.png)
 
+## MessageTimestampFormat for Incoming and Outgoing messages
+
+The `SfChat` allows you to define the format in which timestamps are displayed for outgoing and incoming messages. [IncomingMessageTimestampFormat](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Chat.SfChat.html#Syncfusion_Maui_Chat_SfChat_IncomingMessageTimestampFormat) and [OutgoingMessageTimestampFormat](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Chat.SfChat.html#Syncfusion_Maui_Chat_SfChat_OutgoingMessageTimestampFormat) allow customization of the date and time representation.
```

Comment:
Please use this content: 
The 'SfChat' allows you specify the format in which timestamps are shown for outgoing and incoming messages. The date and time representation can be customised using the IncomingMessageTimestampFormat and OutgoingMessageTimestampFormat properties.

Reply:
Updated as per the requirement